ee/vellum_ee/workflows/display/nodes/vellum/code_execution_node.py

```diff
@@ -40,6 +41,11 @@ def serialize(
                 node_filepath=node_file_path,
                 script_filepath=filepath,
             )
+            if not file_code:
+                raise NodeException(
```

Comment:
[nit] `NodeException` is meant for runtime, let's just use ordinary exception's for now